app/Actions/User/Notify.php

```diff
@@ -22,6 +22,10 @@ public function do(Photo $photo): void
 		$users = new Collection([User::query()->find(0)]);
```

Comment:
We discussed that line already and you had to add some PHPstan exception rules to make it work. Maybe, the best thing we can do here is to be overly explicit like this

```suggestion
		/** @var Collection<int, User> */
		$users = new Collection();
		$users->add(User::query()->sole(['id' => 0]));
```

That should remove the need for any PHPstan exception.

Reply:
See [33bc864](https://github.com/LycheeOrg/Lychee/pull/1469/commits/33bc8649ce7ab16428113bf4ecfe3b1cf7227bdc)